lib/compiler/src/compile.erl

```diff
@@ -719,7 +719,9 @@ core_passes() ->
 
 kernel_passes() ->
     %% Optimizations that must be done after all other optimizations.
-    [{pass,sys_core_bsm},
+    [{pass,sys_core_alias},
+     {iff,dcalias,{listing,"core_alias"}},
```

Comment:
I suggest `dalias` as the name of the option. It is shorter, easier to type, and clear enough.